pkg/chartutil/save.go

```diff
@@ -70,6 +70,12 @@ func SaveDir(c *chart.Chart, dest string) error {
 	// Save files
 	for _, f := range c.Files {
 		n := filepath.Join(outdir, f.TypeUrl)
+
+		d := filepath.Dir(n)
```

Comment:
Is there a way we could do any error checking to see if this is relative to the chart's output directory? We'd want to prevent the user from saving files in, say, /bin because they thought this syntax would work:

```
Files: []*any.Any{
	{TypeUrl: "/usr/share/doc/shahryar.txt", Value: []byte("1,001 Nights")},
},
```

Other than that, I think this looks great. :)

Reply:
You know what, that's something that can be addressed in a follow-up PR instead, since that behaviour can still happen today. Thanks!

Reply:
I can submit a PR for the second case (relative backlinks) once I verify it can be checked. But I can't make the first case happen. From a few quick tests, `filepath.Join(outdir, f.TypeUrl)`, in the absence of `..` in the path, seems to always return a path that falls under `dest`, even if `TypeUrl` begins with a `/`